plugins/hm-stack/inc/api/namespace.php

```diff
@@ -85,3 +109,16 @@ function query_api( $endpoint ) {
 
 	return $data;
 }
+
+/**
+ * Query the /metrics/ API for passthrough Cloudwatch metrics queries.
+ *
+ * @param string $endpoint CloudWatch namespace to query.
+ * @param array $query Query to pass to AWS Cloudwatch API.
+ * @return mixed API response, or null for a bad request,
+ */
+function query_metrics_api( $endpoint, $query ) {
+	$api_base = str_replace( '/applications/', '/metrics/', HM_STACK_API_URL );
```

Comment:
Wondering if we should add a helper to get the right URL and change `HM_STACK_API_URL` to just the common endpoint for all stacks.